DATA_OVERLAY.md

```diff
@@ -36,6 +36,28 @@ Data overlay also allows for two datasets to be selected at the same time. When
 
 <img width="300" alt="image" src="https://user-images.githubusercontent.com/423498/128190231-8b475022-109e-4905-8a01-5e862c397899.png">
 
+### Scaling of gene expression levels from Human Protein Atlas
+
+The gene expression levels used in [Data overlay](https://metabolicatlas.org/documentation#Data-overlay) for the Human-GEM are obtained from the [Human Protein Atlas](https://www.proteinatlas.org/) (HPA), more specifically, the TPM (transcripts per million) data from the file [rna_tissue_hpa.tsv](https://www.proteinatlas.org/download). However, the raw data from HPA is logarithmic, which means the absolute majority of data are clustered in a very narrow bin in the distribution as shown in the figure below.
+
+![Histogram of HPA raw data](./utils/img/hpaRNA-TPM-rawdata-histogram.png)
```

Comment:
It feels that this is not the long-term ideal place for these images, and actually for this `Readme`. I'm in favor of moving the content to the wiki of this repository, to keep the `utils` as utilitarian as possible. If this makes sense to more people, let's have an issue for it.

Reply:
I think the content of the scaling section is in consistent with the rest of sections in this README, maybe because this README looks more like a documentation than a regular README. So we either move the whole readme to the wiki page or keep it as it is. As to the location of the image files, I do see that some repos, such as https://github.com/docker/build-push-action, place the images files in the folder `.github`, maybe we can also adopt it.

Reply:
Hi @mihai-sysbio, should I merge as it is now or you have other opinions?

Reply:
I agree that it should not be in the `/utils` folder. For now I think we could create a new folder at the root level called `/assets` (commonly used in web dev for images and other similar files) and place the `img` folder in there (and update links accordingly). Then we can discuss if we want to move this to the wiki in a separate issue.

Reply:
Good suggestion @e0. I've made the change in 3dc59cf

Reply:
I would have personally gone for `/docs`. Good that this is merged. I've added a note in an existing issue https://github.com/MetabolicAtlas/data-files/issues/9.